datasets/case03/queries/0019.2_selectRunDeepSourceDeepcoaddDeepsrcmatchRefobject.sql

```diff
@@ -0,0 +1,20 @@
+-- Derived from https://dev.lsstcorp.org/trac/wiki/dbPipeQAQueries
+-- Used to test BIT field support
+-- Here ORDER BY fails for unknow reason
+
+SELECT sce.filterName, sce.tract, sce.patch, sro.gMag, sro.ra, sro.decl, sro.isStar,
+       sro.refObjectId, s.id as sourceId,  rom.nSrcMatches, s.flags_pixel_interpolated_center, s.flags_negative,
```

Comment:
remove extra space, maybe make the line a bit shorter (move last element to next line)